src/app/tests/suites/certification/Test_TC_CC_3_1.yaml

```diff
@@ -207,13 +207,22 @@ tests:
                 value: 10000
 
     - label: "TH reads CurrentHue attribute from DUT"
-      PICS: CC.S.F00 && CC.S.A0000
-      command: "readAttribute"
-      attribute: "CurrentHue"
-      response:
-          constraints:
-              minValue: 0
-              maxValue: 15
+      PICS: CC.S.F00 && CC.S.A0000 && PICS_SKIP_SAMPLE_APP
+      verification: |
```

Comment:
why was this changed to userprompt? Is reading attributes with min/max not sufficient? Why?

Reply:
@andy31415 The problem here is that the values count down from 60 -> 0 -> 135, as per the test plan after 10 seconds that values needs to be exactly 0 however this does not happen in real world scenarios as the DUT/Delay command behaviour changes across setups. We had previously added a range check 0-15 [+15% tolerance around the expected value of 0] but -15% is not possible as the expected value is 0 and we currently cannot specify the range checks in descending order.
Please refer this issue:  [416](https://github.com/CHIP-Specifications/chip-test-scripts/issues/416)